README.md

```diff
@@ -87,6 +87,109 @@ with Maven.
 This project uses the [Google java code style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html)
 to keep code clean and consistant.
 
+Maven Plugin
```

Comment:
Do you want this in the `docs/` as well? I actually don't know that maintaining an extensive README makes much sense given that we have the docs -- they are partly historical. I'd suggest treating the README more for developers (i.e. what is this repo? how do I build it? how do I contribute back changes) and leave usage stuff like this to the docs (i.e. I think some of the quickstart stuff could even be removed, in favor of a link to our generated docs).